<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestRoom</class>
 <widget class="QWidget" name="RestRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="RestRoom_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>640</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Game Map</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(180, 4, 4, 255), 
stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);
 
border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 14pt &quot;Candara&quot;;</string>
   </property>
   <property name="text">
    <string>Embark on the Adventure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(180, 4, 4, 255), 
stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);
 
border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 14pt &quot;Candara&quot;;</string>
   </property>
   <property name="text">
    <string>Study Room</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GRYFFINDOR">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>GRYFFINDOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SLYTHERIN">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>SLYTHERIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RAVENCLAW">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>RAVENCLAW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HUFFLEPUFF">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>HUFFLEPUFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>600</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>600</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <zorder>RestRoom_background</zorder>
  <zorder>Back_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>GRYFFINDOR</zorder>
  <zorder>SLYTHERIN</zorder>
  <zorder>RAVENCLAW</zorder>
  <zorder>HUFFLEPUFF</zorder>
  <zorder>Back</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
